.github/workflows/pull-request-checks.yaml

```diff
@@ -4,6 +4,24 @@ on:
     branches: [ develop ]
 
 jobs:
+  check-macos-10_15-make-clang:
+    runs-on: macos-10.15
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Fetch dependencies
+        run: brew install maven flex bison
+      - name: Build using Make
+        run: |
+          make -C src minisat2-download
+          make -C src
```

Comment:
:bulb: It might be worth trying adding `-j2`, to see if adding some parallelism will allow the CI job to complete sooner.